suzieq/poller/worker/services/arpnd.py

```diff
@@ -110,6 +113,44 @@ def _clean_iosxe_data(self, processed_data, raw_data):
     def _clean_ios_data(self, processed_data, raw_data):
         return self._clean_common_ios_data(processed_data, raw_data)
 
+    def _clean_cpgaia_data(self, processed_data, raw_data):
+        for entry in processed_data:
+            if 'state' not in entry:
+                entry['state'] = 'reachable'
+            # FIX oif ?
+        return processed_data
+
+    def _clean_fwsm_data(self, processed_data, raw_data):
+        fakeaddress = ipaddress.ip_address("2.2.2.1")
+        lookup = {}
+
+        # Build a "dns" type lookup table from the "show names" command
+        drop_indices = []
+        for i, entry in enumerate(processed_data):
+            if "alias" in entry:
+                lookup [entry['alias']] = entry['ipAddress']
+                drop_indices.append(i)
+        processed_data = np.delete(processed_data, drop_indices).tolist()
+
+        for entry in processed_data:
+            ip_address = entry['ipAddress']
+            try:
+                # check we have a valid address
+                _ = ipaddress.ip_address(ip_address)
+            except Exception:
+                if ip_address not in lookup:
+                    # This should never be needed....
+                    try:
+                        ip = socket.gethostbyname(ip_address)
+                    except Exception:
+                        ip = fakeaddress # fake it..
```

Comment:
As said above, if we get here I think we should either store the entry as is or discard it. I don't see any point in storing it with fake address. Am I missing something?